lib/services/club/service.dart

```diff
@@ -19,4 +19,21 @@ class ClubService extends BaseService {
       return ServiceResult.failure(response.message);
     }
   }
+
+  Future<ServiceResult<List<ClubModel>>> getTopClubs(
+      {QueryParams? params}) async {
+    final response = await get('/top-clubs', queryParams: params);
```

Comment:
cek dokumentasi, katek query params untuk endpoint ini
```suggestion
  Future<ServiceResult<List<ClubModel>>> getTopClubs() async {
    final response = await get('/top-clubs');
```